viewer/src/public/migration/Cognite3DModel.ts

```diff
@@ -451,6 +451,21 @@ export class Cognite3DModel extends THREE.Object3D implements CogniteModelBase {
     return treeIndices.count;
   }
 
+  /**
+   * Overrrides color for all nodes in the scene.
+   * @param r Red component between 0 and 255.
+   * @param g Green component between 0 and 255.
+   * @param b Blue component between 0 and 255.
+   * @version New in 1.1
```

Comment:
```suggestion
   * @version new in 1.1.0
```

Boomer suggestion here. Since it's not a sentence that is finished with `.` I think it shouldn't start with a capital letter. And let's specify minor `.0` as we do for documentation.